src/home_robot_hw/home_robot_hw/stretch_client/modules/nav.py

```diff
@@ -56,22 +56,6 @@ def at_goal(self) -> bool:
         else:
             return False
 
-    @enforce_enabled
```

Comment:
can we test this with build_3d_map to make sure its still working properly?

Reply:
Will make sure to test it when migrating that script to the new interface.